x/protorev/client/cli/query.go

```diff
@@ -141,12 +141,12 @@ func NewQueryEnabledCmd() (*osmocli.QueryDescriptor, *types.QueryGetProtoRevEnab
 	}, &types.QueryGetProtoRevEnabledRequest{}
 }
 
-// NewQueryPoolWeightsCmd returns the command to query the pool weights of protorev
-func NewQueryPoolWeightsCmd() (*osmocli.QueryDescriptor, *types.QueryGetProtoRevPoolWeightsRequest) {
+// NewQueryInfoByPoolTypeCmd returns the command to query the pool type info of protorev
+func NewQueryInfoByPoolTypeCmd() (*osmocli.QueryDescriptor, *types.QueryGetProtoRevInfoByPoolTypeRequest) {
 	return &osmocli.QueryDescriptor{
-		Use:   "pool-weights",
-		Short: "Query the pool weights used to determine how computationally expensive a route is",
-	}, &types.QueryGetProtoRevPoolWeightsRequest{}
+		Use:   "info-by-pool-type",
+		Short: "Query the pool info used to determine how computationally expensive a route is",
```

Comment:
I assume there is an input right? I think this should be "info-by-pool-type [pool-type]"

Reply:
yea the input is the json file with all of the correct pool info. This isnt meant update by a single pool type (although once the dependencies on a given pool type grow it might be worth migrating to separate message handlers). 

I'll update the PR with the example I used in testing